clientv3/retry.go

```diff
@@ -168,7 +168,7 @@ type retryLeaseClient struct {
 	readRetry retryRPCFunc
 }
 
-// RetryLeaseClient implements a LeaseClient that uses the client's FailFast retry policy.
+// RetryLeaseClient implements a LeaseClient.
```

Comment:
then why do we need the leaseClient at line167?

Reply:
can we rename the readRetry to repeatableRetry? leaseGrant is not really read. 

we will have a retrypolicy: repeatable defined in the future.

Reply:
It has to call `rlc.LeaseClient` where `LeaseClient` is `pb.LeaseClient`?

Reply:
i mean why it is embedded. it is that way so users can call to it directly. but now everything should call through the func we define on retryleaseclient.

Reply:
can we hide the raw client by not embedding it?

Reply:
Right, since retryleaseclient implements the interface, no need to embed. Changed it unexported.